README.md

```diff
@@ -7,7 +7,7 @@ Save the following content to a terraform configuration file yb.tf
 
 ```
 module "yugabyte-db-cluster" {
-  source = ""github.com/YugaByte/terraform-aws-yugabyte"
+  source = "github.com/YugaByte/terraform-aws-yugabyte"
 
   # The name of the cluster to be created.
   cluster_name = "tf-test"
```

Comment:
cluster_name = yb-test

For the ssh_keypair below, can you add a comment explaining what ssh_keypair is? It looks like we expect this to be the name of an AWS ssh keypair

Modify the comment to say 
`# The existing vpc and subnet ids where the nodes should be spawned`

For the AZs can you add a comment below like:
`# Cluster data and metadata will be placed in separate AZs to ensure availability during single AZ failure if 3 AZs are specified.` 
Also explain if a single AZ can be specified or if >= replication_factor AZs have to be specified.

Reply:
Okay, I'll update the suggested changes.